ballerina/tests/generated_client.bal

```diff
@@ -0,0 +1,374 @@
+        _ = check self.persistClients.get("department").runDeleteQuery(deptNo);
+        return 'object;
+    };
+
+    isolated resource function get employees() returns stream<Employee, error?> {
+        stream<anydata, sql:Error?>|Error result = self.persistClients.get("employee").runReadQuery(Employee);
+        if result is Error {
+            return new stream<Employee, Error?>(new EmployeeStream((), result));
+        } else {
+            return new stream<Employee, Error?>(new EmployeeStream(result));
+        }
+    };
+
+    isolated resource function get employees/[string empNo]() returns Employee|error {
+        return (check self.persistClients.get("employee").runReadByKeyQuery(Employee, empNo)).cloneWithType(Employee);
+    };
+
+    isolated resource function post employees(EmployeeInsert[] data) returns string[]|error {
+        _ = check self.persistClients.get("employee").runBatchInsertQuery(data);
+        return from EmployeeInsert inserted in data
+               select inserted.empNo;
+    };
+
+    isolated resource function put employees/[string empNo](EmployeeUpdate data) returns Employee|error {
+        _ = check self.persistClients.get("employee").runUpdateQuery(empNo, data);
+        return self->/employees/[empNo].get();
+    };
+
+    isolated resource function delete employees/[string empNo]() returns Employee|error {
+        Employee 'object = check self->/employees/[empNo].get();
+        _ = check self.persistClients.get("employee").runDeleteQuery(empNo);
+        return 'object;
+    };
+
+    public function close() {
+        
+    }
+}
+
+public class BuildingStream {
+    private stream<anydata, sql:Error?>? anydataStream;
+    private Error? err;
+
+    public isolated function init(stream<anydata, sql:Error?>? anydataStream, Error? err = ()) {
+        self.anydataStream = anydataStream;
+        self.err = err;
+    }
+
+    public isolated function next() returns record {|Building value;|}|Error? {
+        if self.err is error {
+            return self.err;
+        } else if self.anydataStream is stream<anydata, sql:Error?> {
+            var anydataStream = <stream<anydata, sql:Error?>>self.anydataStream;
+            var streamValue = anydataStream.next();
+            if streamValue is () {
+                return streamValue;
+            } else if (streamValue is sql:Error) {
+                return <Error>error(streamValue.message());
+            } else {
+                do {
+                    record {|Building value;|} nextRecord = {value: check streamValue.value.cloneWithType(Building)};
+                    return nextRecord;
+                } on fail error e {
+                    return <Error>e;
+                }
+            }
+        } else {
+            // Unreachable code
+            return ();
+        }
+    }
+
+    public isolated function close() returns Error? {
+        if self.anydataStream is stream<anydata, sql:Error?> {
```

Comment:
Same

Reply:
Fixed in [90235d8](https://github.com/ballerina-platform/module-ballerina-persist/pull/103/commits/90235d817df86c78d05ff4528baba5a18d2a584a)